packages/grid/_modules_/grid/models/api/columnReorderApi.ts

```diff
@@ -1,40 +1,5 @@
-import * as React from 'react';
-import { GridColDef } from '../colDef/gridColDef';
-
+// TODO move to some other place once useGridColumnResize is refactored
 export interface CursorCoordinates {
```

Comment:
Maybe can move this inside `models/` in a new file like `cursorCoordinates.ts` (like the one called `elementSize.ts`)?